### PR TITLE
ISPN-5897 Enhance filter_failed_tests.sh to support compressed logs

### PR DESCRIPTION
**Pull request by danberindei**
https://issues.jboss.org/browse/ISPN-5897

Also update failedTestLogs.sh to require a log file argument and to
use perl more.

### CODE REVIEW

File: bin/failedTestLogs.sh [hunk 1]
```diff
@@ -1,28 +1,30 @@
 #!/bin/sh
```
Comment (tristantarrant) [inline bin/failedTestLogs.sh]: I think things like ${FILE##*.} are not in the basic Bourne shell, so let's make this /bin/bash

Comment (danberindei) [inline bin/failedTestLogs.sh]: Done.

**Comment by tristantarrant** [comment]
Pulling

**Comment by tristantarrant** [comment]
Pushed, thanks

**Pull request closed without merging**